backend/tracim_backend/lib/core/event.py

```diff
@@ -162,6 +162,16 @@ def get_paginated_messages_for_user(
         ).order_by(Message.event_id.desc())
         return get_page(query, per_page=count, page=page_token or False)
 
+    def get_nb_messages(
```

Comment:
`get_messages_count`? I'm unsure if there is a coding rule/a classic usage for this in the backend code.

Reply:
done.